addon/mixins/in-viewport.js

```diff
@@ -34,6 +34,12 @@ export default Mixin.create({
    */
   _debouncedEventHandler: null,
 
+  /**
+   * @property _observerOptions
+   * @default {}
+   */
+  _observerOptions: {},
```

Comment:
Null / computed  returning empty object / initialize on `init`. This could lead to leaking state between objects implementing this mixin.